core/controllers/profile.py

```diff
@@ -76,16 +78,37 @@ def get(self, username):
         user_settings = user_services.get_user_settings_from_username(username)
         if not user_settings:
             raise self.PageNotFoundException
+        
+        user_contributions = user_services.get_user_contributions(
+            user_settings.user_id)
 
         self.values.update({
             'user_bio': user_settings.user_bio,
             'first_contribution_datetime': (
-                utils.get_time_in_millisecs(user_settings.first_contribution_datetime)
+                utils.get_time_in_millisecs(
+                    user_settings.first_contribution_datetime)
                 if user_settings.first_contribution_datetime else None),
             'profile_picture_data_url': user_settings.profile_picture_data_url,
+            'created_explorations_count': len(
+                user_contributions.created_explorations),
+            'edited_explorations_count': len(
+                user_contributions.edited_explorations)
+
         })
         self.render_json(self.values)
 
+    def get_public_contributions(list_explorations):
+        """Pass in created or edited explorations,
+        returns collection with only public explorations in list. 
+        """
+        public_explorations = []
+
+        for exploration_id in list_explorations:
+            if rights_manager.is_exploration_public(exploration):
+                public_explorations.append(exploration_id)
+
+        return public_explorations
```

Comment:
Here, you are returning a list of ids of public explorations to the frontend. Wouldn't the frontend also need additional data? (Though I think you can get that from the summaries.)